### PR TITLE
Remind user to check $PATH after installation.

### LINKED ISSUE #14882: Document that `rustup component add rust-analyzer` just works?
Quite a positive request I suppose, but although the official documentation does not seem to say that (or am I bad at searching?), it would seem like

`rustup component add rust-analyzer`

after a fresh install just works. Maybe it would be worth clearly documenting it, instead of letting people wonder which package they should download or whether they should build from source.

### PR DESCRIPTION
**Pull request by alibektas**
fixes #14882 . I don't think that this is the correct wording to express this but I at least wanted to take the initiative :)

### CODE REVIEW

File: docs/user/manual.adoc [hunk 1]
```diff
@@ -141,6 +141,8 @@ If you're not using Code, you can compile and install only the LSP server:
 $ cargo xtask install --server
 ----
 
+Make sure that `.cargo/bin` is in `$PATH` and precedes paths where `rust-analyzer` may also be installed.
```
Comment (lnicola) [inline docs/user/manual.adoc:144]:
```suggestion
Make sure that `.cargo/bin` is in `$PATH` and precedes paths where `rust-analyzer` may also be installed.
Specifically, `rustup` includes a proxy called `rust-analyzer`, which can cause problems if you're planning to use a source build or even a downloaded binary.
```

**Comment by alibektas** [comment]
> eyes Test was successful, but fast-forwarding failed: 422 Changes must be made through a pull request.

@lnicola should I be worried ? it's my second time merging a pr by myself.

**Comment by lnicola** [comment]
No, it's just a bors thing that happens pretty often.

**Pull request merged**